web/concrete/authentication/concrete/controller.php

```diff
@@ -1,7 +1,8 @@
 <?php
-namespace Concrete\Authentication\Concrete;
+namespace concrete\authentication\concrete;
```

Comment:
@mlocati I think it is.

AFAIK namespaces are case insensitive. This change was suggested by <code>php-cs-fixer</code>.